From fb7764de668abd60c66b0df23abadd1fe36b138e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Sur=C3=BD?= <ondrej@isc.org>
Date: Sun, 4 May 2025 15:19:48 +0200
Subject: [PATCH 6/6] Add extra RSASHA256 2048bit key

---
 bind.keys | 22 +++-------------------
 1 file changed, 3 insertions(+), 19 deletions(-)

diff --git a/bind.keys b/bind.keys
index dffbea5d6b..674947d573 100644
--- a/bind.keys
+++ b/bind.keys
@@ -37,23 +37,7 @@
 # anchor information for the root zone.
 
 trust-anchors {
-        # This key (20326) was published in the root zone in 2017, and
-        # is scheduled to be phased out starting in 2025. It will remain
-        # in the root zone until some time after its successor key has
-        # been activated. It will remain this file until it is removed
-        # from the root zone.
-
-        . initial-key 257 3 8 "AwEAAaz/tAm8yTn4Mfeh5eyI96WSVexTBAvkMgJzkKTOiW1vkIbzxeF3
-                +/4RgWOq7HrxRixHlFlExOLAJr5emLvN7SWXgnLh4+B5xQlNVz8Og8kv
-                ArMtNROxVQuCaSnIDdD5LKyWbRd2n9WGe2R8PzgCmr3EgVLrjyBxWezF
-                0jLHwVN8efS3rCj/EWgvIWgb9tarpVUDK/b58Da+sqqls3eNbuv7pr+e
-                oZG+SrDK6nWeL3c6H5Apxz7LjVc1uTIdsIXxuOLYA4/ilBmSVIzuDWfd
-                RUfhHdY6+cn8HFRm+2hM8AnXGXws9555KrUB5qihylGa8subX2Nn6UwN
-                R1AkUTV74bU=";
-        # This key (38696) will be pre-published in the root zone in 2025
-        # and is scheduled to begin signing in late 2026. At that time,
-        # servers which were already using the old key (20326) should roll
-        # seamlessly to this new one via RFC 5011 rollover.
-        . initial-ds 38696 8 2 "683D2D0ACB8C9B712A1948B27F741219298D0A450D612C483AF444A
-        4C0FB2B16";
+        . initial-key 257 3 8 "AwEAAaz/tAm8yTn4Mfeh5eyI96WSVexTBAvkMgJzkKTOiW1vkIbzxeF3 +/4RgWOq7HrxRixHlFlExOLAJr5emLvN7SWXgnLh4+B5xQlNVz8Og8kv ArMtNROxVQuCaSnIDdD5LKyWbRd2n9WGe2R8PzgCmr3EgVLrjyBxWezF 0jLHwVN8efS3rCj/EWgvIWgb9tarpVUDK/b58Da+sqqls3eNbuv7pr+e oZG+SrDK6nWeL3c6H5Apxz7LjVc1uTIdsIXxuOLYA4/ilBmSVIzuDWfd RUfhHdY6+cn8HFRm+2hM8AnXGXws9555KrUB5qihylGa8subX2Nn6UwN R1AkUTV74bU=";
+        . initial-ds 38696 8 2 "683D2D0ACB8C9B712A1948B27F741219298D0A450D612C483AF444A 4C0FB2B16";
+	. initial-key 257 3 8 "AwEAAcEcT+7Jrsj/nt0EOzP6UgrORVVUuocwSjxvHHx39wnNQDibN0nP 9Q1IcZgPIKIO0WKU8wTl+1D7W3WwklMGeyqg27xIeLtzVW3oehJzSLv0 79H0LnMsxvsDZMLpIiQaQQ9W0f1noIDTCaSJHzZufInhjLN9nHOwPWCg WjtDUbXgVQf0UNJhs4f7LM0P1ffE7OCeKz7RhLkHEjMncC278qq7LU5p ANAzOUZ8xlrJzwQNG7LJ0DBU1iH+IsPFRRPVbUPRKU3d8JNkkHc0YCDv +vPjvJ+LHu9VgClPIMXRPDPqm3ThYYqJ45M+KzCXT4+Av1X6gVrcaUnJ XXkGLWT4DEk=";
 };
-- 
2.43.0

